### PR TITLE
test: add integration test for del

### PR DESCRIPTION
**Pull request by ryota-sakamoto**
*Issue #, if available:*

#143 

*Description of changes:*

I have added the test for `dy del` because the test is nothing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/del.rs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+use predicates::prelude::*; // Used for writing assertions
+
+#[tokio::test]
+async fn test_get_non_existent_table() -> Result<(), Box<dyn std::error::Error>> {
+    let mut c = util::setup().await?;
+    let cmd = c.args(&[
+        "--region",
+        "local",
+        "--table",
+        "dummy-table-doent-exist",
```
[the remark this answers is not in the document]
Comment (ryota-sakamoto) [reply tests/del.rs:29]: It is just result of copy and paste so I will fix all of the typo.

File: tests/util/mod.rs [hunk 2]
```diff
@@ -150,6 +150,22 @@ pub async fn create_temporary_table(keys: Vec<&str>) -> Result<String, Box<dyn s
...
 }
 
+pub async fn create_temporary_table_with_items(
+    keys: Vec<&str>,
+    items: Vec<Vec<&str>>,
+) -> Result<String, Box<dyn std::error::Error>> {
+    let table_name = create_temporary_table(keys).await?;
+
+    for item in items {
+        let mut c = setup().await?;
```
[the remark this answers is not in the document]
Comment (ryota-sakamoto) [reply tests/util/mod.rs:160]: Yes it is correct, but from my opinion, I want to use the function setup to unify in all of test.

File: tests/del.rs [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        "Successfully deleted an item from the table '{}'.\n",
+        table_name
+    ));
+
+    let mut c = util::setup().await?;
+    let scan_cmd = c.args(&["--region", "local", "--table", &table_name, "scan"]);
+    scan_cmd
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("pk  attributes\nb"));
```
[the remark this answers is not in the document]
Comment (ryota-sakamoto) [reply tests/del.rs:71]: Cool. I didn't know that. I will use it.

[the remark this answers is not in the document]
Comment (ryota-sakamoto) [reply tests/del.rs]: [The result](https://github.com/awslabs/dynein/actions/runs/5894474398/job/15988183383#step:8:144) is example of diff

**Comment by StoneDot** [comment]
All discussions are resolved. And It seems good to me. I will merge this PR. Thank you for your patience of my delay in the review.

**Review by StoneDot — APPROVED** [review]

**Pull request merged**